packages/models-library/src/models_library/services_metadata_published.py

```diff
@@ -149,6 +149,18 @@ class ServiceMetaDataPublished(ServiceKeyVersion, ServiceBaseDisplay):
         alias="progress_regexp",
         description="regexp pattern for detecting computational service's progress",
     )
+    inputs_path: str | None = Field(
+        None,
+        description="if this is present, the service is a modern style dv2 dynamic service",
```

Comment:
If so, I am surprised that was not already here ...
CAREFUL with changes in this class since it is coupled in many projects. Specifically  with the `.osparc/metadata.yml` file used in OOIL

https://github.com/itisfoundation/osparc-simcore/blob/c37d45b4ad9987eb30c6c3b1958f7d3e908354d5/packages/service-integration/src/service_integration/osparc_config.py#L99-L104